### ISSUE TITLE
[Issue] Cannot load 2d models that use builtin/generated.

### ISSUE THREAD

**Issue by ououn** [JannisX11/blockbench#1070]
**Issue Description:**
"This file is a child of builtin/generated and does not contain a model"
This warning appear when trying to load Java block/item models that use "builtin/generated" as the parent.

**Background story:**
In Java, most 2d flat looking models we see does not contain any elements, instead it use a parent called item/generated (or item/handheld, item/handheld_rod, etc ?) that contain display information and another parent that do the 2d generating job, which is builtin/generated.

**Why is this an issue?**
It is possible to have a json model that straight ahead use builtin/generated as parent, but right now we cannot load model that use such parent in blockbench, thus make it impossible to edit the display for such model in blockbench.

**Comment by JannisX11** [comment]
Blockbench loads generated models if the model is valid. Valid generated item models must have one of the parents you listed above, a texture with the key "layer0", and can't have any elements. I cannot reproduce any issues with valid models.

**Comment by ououn** [comment]
Let me rephrase the issue...Most valid model one can get from default assets should be using one of those json model in the item folder, so most of them are **item/something**. However the **builtin/generated**, which most 2d item model don't directly use as their parent,  doesn't load in blockbench.

Below is an item model that look and display exactly like diamond sword, load just fine ingame, so the model is valid, but not in blockbench. Tested in 3.9.3 and 4.0.0 beta-1

```json
{
	"parent": "minecraft:builtin/generated",
	"gui_light": "front",
	"textures": {
		"layer0": "minecraft:item/diamond_sword"
	},
	"display": {
		"ground": {
			"rotation": [ 0, 0, 0 ],
			"translation": [ 0, 2, 0 ],
			"scale": [ 0.5, 0.5, 0.5 ]
		},
		"head": {
			"rotation": [ 0, 180, 0 ],
			"translation": [ 0, 13, 7 ],
			"scale": [ 1, 1, 1 ]
		},
		"thirdperson_righthand": {
			"rotation": [ 0, -90, 55 ],
			"translation": [ 0, 4.0, 0.5 ],
			"scale": [ 0.85, 0.85, 0.85 ]
		},
		"thirdperson_lefthand": {
			"rotation": [ 0, 90, -55 ],
			"translation": [ 0, 4.0, 0.5 ],
			"scale": [ 0.85, 0.85, 0.85 ]
		},
		"firstperson_righthand": {
			"rotation": [ 0, -90, 25 ],
			"translation": [ 1.13, 3.2, 1.13 ],
			"scale": [ 0.68, 0.68, 0.68 ]
		},
		"firstperson_lefthand": {
			"rotation": [ 0, 90, -25 ],
			"translation": [ 1.13, 3.2, 1.13 ],
			"scale": [ 0.68, 0.68, 0.68 ]
		},
		"fixed": {
			"rotation": [ 0, 180, 0 ],
			"scale": [ 1, 1, 1 ]
		}
	}
}
```
